sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_analyze_action_healthcare_async.py

```diff
@@ -0,0 +1,126 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_action_healthcare_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to submit a collection of text documents for analysis, which uses the
+    AnalyzeHealthcareEntitiesAction and RecognizePiiEntitiesAction to recognize healthcare entities,
```

Comment:
Added a sample that covers 2 actions since I read that that is our recommendation when using actions.